configs/ground_truth_mapping.json

```diff
@@ -0,0 +1,326 @@
+{
+  "mac_1_2": [
```

Comment:
maybe this shared config file was a bad idea.....

Reply:
I kind of see why you might think that... but we're never looking at this right so it's maybe okay?